docs/source/ops/connect/auth0.rst

```diff
@@ -695,7 +695,7 @@ production ledger (minus persistence).
     daml sandbox --ledgerid %%LEDGER_ID%% \
                  --auth-jwt-rs256-jwks https://%%AUTH0_DOMAIN%%/.well-known/jwks.json \
                  --implicit-party-allocation false \
```

Comment:
this is still wrong, the other args don’t work either but not more wrong than before :shrug: Not for this PR to fix

Reply:
Indeed, fixing it takes a bit more than just changing the command line: the equivalent path for canton sandbox needs to be described. It is easier to do it when the old text is still present for reference.